lib/Transforms/Scalar/UnifyAssumes.cc

```diff
@@ -135,6 +181,34 @@ bool UnifyAssumesPass::runOnFunction(Function &F) {
   return false;
 }
 
+void UnifyAssumesPass::processAssertInst(CallInst &CI, AllocaInst &flag) {
+  BasicBlock *bb = CI.getParent();
+  Module *M = bb->getParent()->getParent();
+  assert(bb);
+  assert(M);
+
+  auto &SBI = getAnalysis<seahorn::SeaBuiltinsInfoWrapperPass>().getSBI();
+  llvm::CallSite CS(&CI);
+  IRBuilder<> B(bb);
+  B.SetInsertPoint(&CI);
+  auto ante = B.CreateLoad(&flag);
```

Comment:
proper terminology should be `antecedent` and `consequent` not `antecedent` and `condition`

Reply:
fixed